app/models/plan.rb

```diff
@@ -152,14 +152,15 @@ def excluded_actions_for(benchmark_indicator)
   def current_and_target_scores
     indicators = BenchmarkIndicator.all.pluck(:id)
 
-    indicators.reduce({}) do |sag, i|
-      score = score_value_for(assessment_indicator: i)
+    indicators.reduce({}) do |scores, indicator_id|
+      current = score_value_for(assessment_indicator: indicator_id)
```

Comment:
unless I'm missing something, isn't the method `score_value_for` only present on `PlanBuilder`?

Reply:
also, isn't this `indicator_id` a `BenchmarkIndicator` id, and not an `AssessmentIndicator` id? that's probably not going to return the score that you are looking for if so :)

Reply:
> unless I'm missing something, isn't the method `score_value_for` only present on `PlanBuilder`?

Yes, but `PlanBuilder` is included in the Plan model

Reply:
> also, isn't this `indicator_id` a `BenchmarkIndicator` id, and not an `AssessmentIndicator` id? that's probably not going to return the score that you are looking for if so :)

Riiiiight... fixing that